### PR TITLE
Add accounts delete subcommand

### PR DESCRIPTION
**Pull request by tirthct**
**Changed**
- Add accounts delete subcommand
- Modified readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -331,3 +331,18 @@ Use the `subscriptionRoleBinding` subcommand to remove a role binding from an ac
 * For the given account, delete a role binding created at application level using the roleID `ocm support delete applicationRoleBinding [accountID] [roleID]`
 * For the given account, delete a role binding created at organization level using the roleID `ocm support delete organizationRoleBinding [accountID] [orgID] [roleID]`
 * For the given account, delete a role binding created at subscription level using the roleID `ocm support delete subscriptionRoleBinding [accountID] [subscriptionID] [roleID]`
+
+#### Deleting registry credentials
```
Comment (cristianoveiga) [inline README.md:335]: Deleting Account?

File: README.md [hunk 2]
```diff
@@ -331,3 +331,18 @@ Use the `subscriptionRoleBinding` subcommand to remove a role binding from an ac
...
+The following flags are available for `accounts delete`:
+
+```
+--dry-run                    If false, deletes the account for the given account ID, defaults to true.
+-h, --help                   help for delete
+```
+
+#### Examples
+
+* Delete a specific account `ocm support delete accounts [accountID]`ßß
```
Comment (cristianoveiga) [inline README.md:348]: what are the `ßß` chars at the end?

File: cmd/ocm-support/delete/cmd.go [hunk 3]
```diff
@@ -1,6 +1,7 @@
 package delete
 
 import (
+	"github.com/openshift-online/ocm-support-cli/cmd/ocm-support/delete/accounts"
```
Comment (cristianoveiga) [inline cmd/ocm-support/delete/cmd.go:4]: nitpick: review import order.

File: README.md [hunk 4]
```diff
@@ -331,3 +331,18 @@ Use the `subscriptionRoleBinding` subcommand to remove a role binding from an ac
 * For the given account, delete a role binding created at application level using the roleID `ocm support delete applicationRoleBinding [accountID] [roleID]`
 * For the given account, delete a role binding created at organization level using the roleID `ocm support delete organizationRoleBinding [accountID] [orgID] [roleID]`
 * For the given account, delete a role binding created at subscription level using the roleID `ocm support delete subscriptionRoleBinding [accountID] [subscriptionID] [roleID]`
+
+#### Deleting an account
+
+Use the `accounts` subcommand to to delete a specific account for the passed accountID.
```
Comment (cristianoveiga) [inline README.md:337]: `accounts delete` ?

File: README.md [hunk 5]
```diff
@@ -331,3 +331,18 @@ Use the `subscriptionRoleBinding` subcommand to remove a role binding from an ac
...
+The following flags are available for `accounts delete`:
+
+```
+--dry-run                    If false, deletes the account for the given account ID, defaults to true.
+-h, --help                   help for delete
+```
+
+#### Examples
+
+* Delete a specific account `ocm support delete accounts [accountID]`
```
Comment (cristianoveiga) [inline README.md:348]: is it ocm support delete account ?

Comment (tirthct) [reply README.md]: Changed to account

File: README.md (hunk 4 above)
Comment (tirthct) [reply README.md]: Changed to account

**Pull request closed without merging**